examples/ecmascript/package.json

```diff
@@ -40,5 +40,10 @@
     "src",
     "!**/*.test.*",
     "!**/testdata/**"
-  ]
+  ],
+  "dependencies": {
+    "@pkgjs/parseargs": "^0.11.0",
+    "ecmarkup": "^17.0.0",
```

Comment:
Same, is this used/required?

Reply:
This one is detritus. I'll remove & repush :)